substrate/frame/revive/fixtures/contracts/Caller.sol

```diff
@@ -65,4 +67,22 @@ contract Caller {
             }
         }
     }
+
+    function callPartialGas(address _callee, bytes memory _data, uint64 _gasDivisor, uint8 _callType)
```

Comment:
We should probably use a 
```solidity 
        enum CallType {
            Call,
            StaticCall,
            DelegateCall
        }
```

that then generate the enum in Rust as well (Caller::CallType::Call), making both the fixture and calling code cleaner